lib/spring/client/run.rb

```diff
@@ -138,6 +140,8 @@ def run_command(client, application)
         if pid && !pid.empty?
           log "got pid: #{pid}"
 
+          puts "Running Spring [client] #{pid}" if Spring.verbose_output
```

Comment:
I'd rephrase this:

```
Running via Spring preloader in process 1234
```

* I'm not sure the `[client]` bit has any meaning to somebody who doesn't know about the details of Spring's implementation
* The "Spring preloader" bit can be put into a search engine to figure out what Spring is (for users who have installed Rails out-the-box and who don't already know)